components/widgets/forest-change/glad-alerts-daily/index.js

```diff
@@ -0,0 +1,290 @@
+import { fetchGLADLatest, fetchHistoricalGladAlerts } from 'services/analysis-cached';
+
+import {
+  POLITICAL_BOUNDARIES_DATASET,
+  GLAD_DEFORESTATION_ALERTS_DATASET
+} from 'data/layers-datasets';
+import {
+  DISPUTED_POLITICAL_BOUNDARIES,
+  POLITICAL_BOUNDARIES,
+  GLAD_ALERTS
+} from 'data/layers';
+
+import getWidgetProps from './selectors';
+
+export default {
+  widget: 'gladAlertsDaily',
+  title: 'Glad Alerts Count in {location}',
+  large: true,
+  refetchKeys: ['dataset', 'forestType', 'landCategory', 'confidence'],
+  settingsConfig: [
+
+  ],
+  visible: ['dashboard', 'analysis'],
+  types: ['country', 'geostore', 'wdpa', 'aoi', 'use'],
+  categories: ['summary', 'forest-change'],
+  admins: ['adm0', 'adm1', 'adm2'],
+  chartType: 'composedChart',
+  source: 'gadm',
+  dataType: 'glad',
+  colors: 'loss',
+  metaKey: 'widget_deforestation_graph',
+  settings: {
+    dataset: 'glad',
+    minDate: '2000-01-01',
+    confidence: ''
+  },
+  datasets: [
+    {
+      dataset: POLITICAL_BOUNDARIES_DATASET,
+      layers: [DISPUTED_POLITICAL_BOUNDARIES, POLITICAL_BOUNDARIES],
+      boundary: true,
+    },
+    {
+      dataset: GLAD_DEFORESTATION_ALERTS_DATASET,
+      layers: [GLAD_ALERTS],
+    },
+  ],
+  sentences: {
+    initial:
+      'Between {start_date} and {end_date} {location} experienced a total of {total_alerts} glad alerts',
+    withInd:
+      'Between {start_date} and {end_date} {location} experienced a total of {total_alerts} glad alerts within {indicator}',
+    highConfidence: ', considering <b>high confidence</b> alerts only.',
```

Comment:
not needed as confidence isnt part of glads

Reply:
removed